### PR TITLE
AES CTR benchmark

### PR DESCRIPTION
**Pull request by RonEld**
Add Benchmark support for AES CTR

### CODE REVIEW

File: benchmark/main.cpp [hunk 1]
```diff
@@ -496,11 +498,35 @@ static int benchmark( int argc, char *argv[] )
             mbedtls_aes_setkey_enc( &aes, tmp, keysize );
 
             TIME_AND_TSC( title,
-                mbedtls_aes_crypt_cbc( &aes, MBEDTLS_AES_ENCRYPT, BUFSIZE, tmp, buf, buf ) );
+                mbedtls_aes_crypt_cbc( &aes, MBEDTLS_AES_ENCRYPT,BUFSIZE , tmp, buf, buf ) );
```
Comment (RonEld) [inline benchmark/main.cpp]: copy\paste error, I will fix it

**Comment by RonEld** [comment]
@andresag01 I fixed according to your comments

**Comment by andresag01** [comment]
@RonEld: Thanks for fixing the changes. Before approving and merging, it would be nice to hear @mazimkhan's opinion to see whether we need to update the log file as well considering that `MBEDTLS_CIPHER_MODE_CTR` is not enabled by default in the library.

**Comment by andresag01** [comment]
I have cheked the log and we are missing the prints for the other features that are turned off  y default, so I do not think we need to add AES-CTR to the test log for `htrun`

**Comment by andresag01** [comment]
@sbutcher-arm, @yanesca: I will wait for additional reviews for this PR before merging.

**Comment by andresag01** [comment]
@yanesca and @gilles-peskine-arm: Could one of you please take a look at this?

**Comment by andresag01** [comment]
Merging this change. All the expected tests pass in the CI.

**Pull request merged**